### ISSUE TITLE
logFile duplicates

### ISSUE THREAD

**Issue by landge** [cbedetti/Dcm2Bids#29]
Hi,
I'm processing a dataset with ca. 400 MR-scans with dcm2bids.
To get an overview of the number of MATCH/NO MATCHes I'm parsing the log-files.
The problem is that dcm2bids writes a lot of logfiles with duplicate entries. So it's never totally
clear which logfile to use for counting/parsing.
When I look at your code it seems that dcm2bids should write one logfile per scan. Is it possible that
the filehandler in the logger is not closed properly or something like that? If tried to test it, but I don't really know how to access the Filehandler in the logger at the end of the run() call.
Have you any ideas what could cause this behaviour?

Thanks

**Comment by cbedetti** [comment]
You're right, the issue is there when dcm2bids is not run from the script.

Something like that should be implemented in `dcm2bids.logging` and called from `Dcm2bids.run()` :
https://stackoverflow.com/questions/5296130/restart-logging-to-a-new-file-python

**Comment by arnaudbore** [comment]
Does not seem to be relevant anymore.
